python/custom_test_files.py

```diff
@@ -0,0 +1,189 @@
+#The following code imports the required packages.
+import math
+import json
+import os
+import pandas as pd
+import random
+import string
+import time
+import yaml
+import shutil as sh
+
+from lar_generator import lar_gen #Imports lar_gen class. 
+lg = lar_gen() #Instantiates lar_gen class as lg. 
+
+class Custom_Test_Files(object):
+    
+    """Returns a Custom_Test_Files object with a set of functions and parameters to create submission files
+    with a specified number of rows."""
+    
+    def __init__(self, filename):
+        self.filename = filename #Instantiates the class with an existing file. 
+        print("Custom_Test_Files Object Instantiated.")
+            
+    def create_file(self, row_count=None, save_file=None):
+        """
+        Creates a new custom file, passing in a row count and filepath to save the created file. 
+    
+        """
+        ts_file = self.separate_ts() #Stores the file path for TS data as ts_file. 
+        lar_file = self.separate_lar() #Stores the file path for LAR data as lar_file. 
+        #Uses the get_rows function to create a dataframe of LAR and to add on the specified number of rows. 
+        df = self.get_rows(row_count=row_count, lar_file=lar_file) 
+        #Uses the assign rows function to create unique ULI's. 
+        df = self.assign_uli(df=df, ts_file=ts_file)
+        #Places the dataframe of LAR in a file with a TS row. 
+        self.lar_to_hmda_file(ts_file=ts_file, df=df, save_file=save_file)
+    
+    def separate_ts(self):  
+        """
```

Comment:
It looks like separate_ts and separate_lar are duplicative. Let's explore the idea of having a single function that does both jobs.